frontend/src/components/content-grid/preview/generic.vue

```diff
@@ -39,5 +51,15 @@ c-anchor-root.group.w-full(class="max-w-[416px]")
         | {{ viewership }} {{ viewershipName + (viewership === 1 ? '' : 's') }}
         | •
         | {{ dateToElapsedTimeString(beginAt) }}
-    c-icon.text-slate-300(v-if="icon" :name="icon")
+    .flex.flex-col.justify-between.items-center
+      template(v-if="kebabMenu")
+        c-dropdown.flex.justify-center(class="[&_>_ul]:w-fit [&_>_ul]:right-0 [&_>_ul]:top-full")
```

Comment:
Can we incorporate these ul styles into the dropdown (either always enabled or enabled through props)? In general styles should not mention elements that aren't immediately visible in the template, otherwise the code becomes too coupled and hard to maintain.

Also, why do we need the flex styles?